brigade-worker/src/require.ts

```diff
@@ -16,10 +18,20 @@ class Pkg {
 }
 
 export function overridingRequire(pkg: string): any {
+  return require(getOverriddenPackage(pkg));
+}
+
+export function getOverriddenPackage(pkg: string): string {
   for (let p of pkgOverrides) {
     if (p.name == pkg) {
-      return require(p.override);
+      return p.override;
+    }
+
+    // we want to intercept the loading of relative modules in the repo
+    // and and override them with the absolute path inside the worker pod
+    if ((!path.isAbsolute(pkg)) && (pkg.includes("./"))) {
```

Comment:
shouldn't it be `path.startsWith`?

Reply:
Thanks for catching that, updating.

Reply:
Although on second thought, here's a scenario I haven't tested yet - 
In the context of #766, with `brigade.js` is in a custom location, say in a directory called `brigade`, and I want to load a module from `../js/my-module`, both our assumptions fail:

- the path does not start with `./`;
- the result of that path manipulation is not correct - it becomes `"./vcs/js/my-module"`, which is not the correct absolute path.

Reply:
hm.... trying to think of something else as well. brigade.js references fileA.js, which references fileB.js. Do we need to do any changes in fileA or fileB?